### PR TITLE
Makes tactical turtlenecks extra tactical by adding tactical fire immunity, tactically

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: Syndicate Advanced Tactics Division Update
tweak: Tactical turtlenecks now have tactical fire immunity
/:cl:

beat you to it @yackemflam 

does kevinz go out of his way to thumbs down all my prs

### CODE REVIEW

**Comment by yackemflam** [comment]
But the tacticool is supposed to be worthless.

**Comment by Tacolizard** [comment]
@yackemflam isn't the tacticool the one you get from the hacked clothing vendor?

**Comment by Militaires** [comment]
@Tacolizard @yackemflam 

the tacticool is referred to as tactifool in the dmi for a reason, yes. that is correct.

**Comment by nullbear** [comment]
you also get it as an arcade machine reward

**Comment by nullbear** [comment]
You could say that this makes tacticool turtlenecks.... tacticooling. B)

**Comment by GunHog** [comment]
Make sure that this is only the Nuke Op version, not the ones you can get easily on the station.

**Comment by nullbear** [comment]
there are like a whopping total of 3 available on the station, and it's not exactly round-ending heat resistance.

**Comment by Tacolizard** [comment]
@nullbear exactly, there's not that many and they don't even cover the whole body, so you still take a good deal of damage.

**Comment by Tacolizard** [comment]
@yackemflam the tacticool does also have the same fire and acid protection as the normal tactical turtleneck.

**Comment by kevinz000** [comment]
Fine have a 👍

**Comment by PKPenguin321** [comment]
that is a bug tacticool is supposed to be useless

**Comment by Bawhoppen** [comment]
definition of powercreep :+1:

File: code/modules/clothing/under/syndicate.dm [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 	alt_covers_chest = 1
 
 /obj/item/clothing/under/syndicate/tacticool
 	name = "tacticool turtleneck"
-	desc = "Just looking at it makes you want to buy an SKS, go into the woods, and -operate-."
+	desc = "Just looking at this highly tactical heat-insulated turtleneck makes you want to buy an SKS, go into the woods, and -operate-."
 	icon_state = "tactifool"
 	item_state = "bl_suit"
 	item_color = "tactifool"
+	max_heat_protection_temperature = FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT
```
Comment (Core0verload) [inline code/modules/clothing/under/syndicate.dm]: Just remove it, you don't need to repeat it in each subtype.

**Comment by phil235** [comment]
Title is confusing, the PR adds heat protection, not fire immunity.

**Comment by kevinz000** [comment]
@Tacolizard  with new resistance flags/clothing damage shit this'll still burn.

**Comment by yackemflam** [comment]
Power creep the clothes pr

**Comment by yackemflam** [comment]
With this or I can make myself fire immune until my clothes break.
Firefighter helmet - head
Tacticool - body + limbs
Black gloves - hands
Winter boots - feet
And still have room for the armor slot.

**Comment by Tacolizard** [comment]
@yackemflam i fail to see how that is even slightly overpowered.

1. requires clothes from all over the place
2. one of the items is job exclusive
3. the clothes will still break
4. very rarely would someone go through so much effort for fire protection
5. anyone should be capable of defending themselves from anything, given enough effort

**Comment by optimumtact** [comment]
Remove the heat protection from the tacticool turtleneck.

**Comment by kevinz000** [comment]
Oh shit oranges has spoken

File: code/modules/clothing/under/syndicate.dm [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 	name = "tactical turtleneck"
-	desc = "A non-descript and slightly suspicious looking turtleneck with digital camouflage cargo pants."
+	desc = "A suspicious looking turtleneck with digital camouflage cargo pants. The tag says it's made of 100% asbestos."
 	icon_state = "syndicate"
 	item_state = "bl_suit"
 	item_color = "syndicate"
 	has_sensor = 0
-	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 40)
+	max_heat_protection_temperature = FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT
+	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0, fire = 200, acid = 40)
```
Comment (optimumtact) [inline code/modules/clothing/under/syndicate.dm]: twice the legal amounts of protection bois

Comment (optimumtact) [inline code/modules/clothing/under/syndicate.dm]: it's just heat proof and still burns

you can't explain that

**Pull request closed without merging**